internal/services/environment_settings/datasource_environment_settings.go

```diff
@@ -147,6 +147,11 @@ func (d *EnvironmentSettingsDataSource) Schema(ctx context.Context, req datasour
 								MarkdownDescription: "Is read audit enabled",
 								Optional:            true,
 							},
+							"log_retention_period_in_days": schema.Int64Attribute{
+								Description:         "log_retention_period_in_days",
```

Comment:
You can remove all the Descriptions and leave only MarkdownDescriptions as they are redundand.